printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md

```diff
@@ -0,0 +1,8 @@
+# LGX stealthburner connector cover
+
+I was searching for connector cover that could fit my stealthburner LGX toolhead.
+I could not find a cover that would fit my toolhead, so I redesigned [craxoor](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) Afterburner PCB Cover. 
```

Comment:
Just a suggestion, maybe consider rewriting this as: 

```This is a remix of [craxoor's Afterburner PCB Cover](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) designed to work with the Stealthburner and Bondtech's LGX Extruder.```

Reply:
Thank you. This sounds a lot better